### PR TITLE
[RDY] lsp: fix on_attach signature documentation

### PR DESCRIPTION
**Pull request by mjlbach**
related to https://github.com/neovim/nvim-lspconfig/issues/63

also see https://github.com/neovim/nvim-lspconfig/pull/647

### CODE REVIEW

File: runtime/doc/lsp.txt [hunk 1]
```diff
@@ -639,8 +639,8 @@ client()                                                      *vim.lsp.client*
                     automatically escalate and force shutdown.
                   • is_stopped() Checks whether a client is stopped. Returns:
                     true if the client is fully stopped.
-                  • on_attach(bufnr) Runs the on_attach function from the
-                    client's config if it was defined.
+                  • on_attach(client, bufnr) Runs the on_attach function from the
+                    client's config if it was defined. Useful for doing buffer-local setup.
```
Comment (teto) [inline runtime/doc/lsp.txt:643]: isn't it better without `doing` ?

Comment (mjlbach) [reply runtime/doc/lsp.txt]: Yes (I just copied the verbage from the old lspconfig readme).

Comment (mjlbach) [reply runtime/doc/lsp.txt]: Fixed

**Comment by mjlbach** [comment]
Wait no it's not

**Comment by mjlbach** [comment]
Now it is (fixed vimdoc gen)

**Pull request merged**